string className
sensor_msgs/Image depthImg
bounding_box box
